monai/engines/trainer.py

```
@@ -36,7 +36,14 @@
     Metric, _ = optional_import("ignite.metrics", IgniteInfo.OPT_IMPORT_VERSION, min_version, "Metric")
     EventEnum, _ = optional_import("ignite.engine", IgniteInfo.OPT_IMPORT_VERSION, min_version, "EventEnum")
 
-__all__ = ["Trainer", "SupervisedTrainer", "GanTrainer", "AdversarialTrainer"]
+__all__ = [
+    "Trainer",
+    "SingleNetworkTrainer",
+    "SupervisedTrainer",
+    "DualNetworkTrainer",
+    "GanTrainer",
+    "AdversarialTrainer",
+]
```

Ruff (0.15.18)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)